quickstarts/newrelic-ingest-metric/config.yml

```
@@ -0,0 +1,51 @@
+# Name of the quickstart (required)
+name: newrelic-ingest-metric
+
+# Displayed in the UI (required)
+title: New Relic Ingest Metric
+
+# Long-form description of the quickstart (required)
+description: |
+  This quickstart provides a synthetic monitor (and associated dashboard) that will regularly query ingest for each data type (APM Events, Infrastructure Processes, etc.) and post it to the Metric API as a custom metric `newrelic-ingest-metric`. This means that you can query and analyze ingest across your account for 13 months by default, instead of 8 days for events, 30 days for  Logs, etc. The custom metric also includes a facet for each data type, so that you can see, for example, ingest for Mobile Events by appName or Logging by labels.app. The following parameters have default values but are all configurable:
+  - ingest interval (hourly, daily, weekly)
+  - facet for each data type
+  - name of custom metric
+  - addition of custom events
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Use this script to set up a synthetic monitor that:
+  1. Queries ingestion by data source, faceted by the attribute of your choice (e.g., APM Events by appName)
+  2. Transforms the output into a metric that is pushed to the Metric API
+  3. Can be queried for consumption analysis / dashboards for 13 months
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Nora Shannon Johnson
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - ingest
+  - ingestion
+  - custom metric
+  - synthetic monitor
+  - consumption
+  - billing
+  - facet
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - newrelic-ingest-metric
+
+documentation:
+  - name: nr1-ingest-metric-script (github)
+    url: https://github.com/newrelic-experimental/nr1-ingest-metric-script
+    description: Instructions for updating and activating the synthetic monitor that generates the newrelic.ingest custom metric
+
+# Content / Design
+logo: logo.png
```

```suggestion
icon: logo.png
```

We recently changed up how we do logos in our config files. We moved the 'logo' keyword to 'icon'